packages/apply/index.test.ts

```diff
@@ -1090,3 +1090,143 @@ describe("package file apply bump", () => {
     restoreConsole();
   });
 });
+
+describe("packge file applies preview bump", () => {
+  let restoreConsole: RestoreConsole;
+  beforeEach(() => {
+    restoreConsole = mockConsole(["log", "dir"]);
+  });
+  afterEach(() => {
+    restoreConsole();
+  });
+
+  it('bumps single js json', function* () {
+    const jsonFolder = f.copy("pkg.js-single-json"); // 0.5.9
+
+    const commands = [
+      {
+        dependencies: undefined,
+        manager: "javascript",
+        path: "./",
+        pkg: "js-single-json-fixture",
+        type: "minor",
+        parents: [],
+      },
+    ];
+
+    const config = {
+      packages: {
+        "js-single-json-fixture": {
+          path: "./",
+          manager: "javascript",
+        },
+      },
+    };
+
+    //@ts-ignore
+    yield apply({ commands, config, cwd: jsonFolder, previewVersion: 'branch-name.12345' });
+    //@ts-ignore
+    const modifiedVFile = yield toVFile.read(
+      jsonFolder + "/package.json",
+      "utf-8"
+    );
+    expect(modifiedVFile.contents).toBe(
+      "{\n" +
+        '  "private": true,\n' +
+        '  "name": "js-single-json-fixture",\n' +
+        '  "description": "A single package at the root. No monorepo setup.",\n' +
+        '  "version": "0.5.9-branch-name.12345"\n' +
+        "}\n"
+    );
+
+    expect({
```

Comment:
Looking at the snapshot, `consoleDir` and `consoleLog` don't have any data, Is that expected? I presume there would be some output here.

Reply:
The reason why there's no console log is because it's been omitted but I just added it [here](https://github.com/jbolda/covector/pull/187/commits/7f11f9ecfba0f2405922fac6b604d5e0a1411b67)